src/exojax/spec/atomll.py

```diff
@@ -0,0 +1,408 @@
+import numpy as np
+from exojax.spec import atomllapi
+#import jax.numpy as jnp
+
+def Sij0(A, gupper, nu_lines, elower, QTref_284, QTmask, Irwin=False):
+    """Reference Line Strength in Tref=296K, S0.
+
+    Note:
+       Tref=296K
+
+    Args:.
+       A: Einstein coefficient (s-1)
+       gupper: the upper state statistical weight
+       nu_lines: line center wavenumber (cm-1)
+       elower: elower
+       QTref_284: partition function Q(Tref)
+       QTmask: mask to identify a rows of QTref_284 to apply for each line
+       Irwin: if True(1), the partition functions of Irwin1981 is used, otherwise those of Barklem&Collet2016
+
+    Returns:
+       Sij(T): Line strength (cm)
+
+    """
+    ccgs=29979245800.0 #[cm/s]
```

Comment:
We can import physical constants from [utils.constants](https://github.com/HajimeKawahara/exojax/blob/develop/src/exojax/utils/constants.py).

```python
from exojax.utils.constants import ccgs, hcperk
```
you can add other constants if you do not find them in exojax.utils.constants.